src/cpp/fastdds/builtin/type_lookup_service/TypeLookupReplyListener.cpp

```diff
@@ -18,7 +18,6 @@
  */
 
 #include <fastdds/builtin/type_lookup_service/TypeLookupReplyListener.hpp>
-
```

Comment:
Please, keep a blank line between the header corresponding to this source file and the other includes as per the include header order style guideline.

Reply:
added in [8020bf2](https://github.com/eProsima/Fast-DDS/pull/4240/commits/8020bf2136b0ed793a5e093b26fc59595dbd7b26)